piker/brokers/kraken.py

```diff
@@ -772,13 +756,21 @@ async def subscribe(ws: wsproto.WSConnection, token: str):
                             status='filled',
                             filled=float(trade.size),
                             reason='Order filled by kraken',
-                            # remaining='' # TODO: not sure what to do here.
                             broker_details={
                                 'name': 'kraken',
                                 'broker_time': trade.broker_time
-                            }
+                            },
+
+                            # TODO: figure out if kraken gives a count
```

Comment:
looks good so far.

I think kraken will send the individual trades made, even if the order was complete over several trades. does this mean we need to handle the decrement ourselves?

Reply:
yeah i think so. doesn't have to be solved here which is why i made #296 to address it more specifically. unfortunately we'll likely have to test with real clearings?